playwright/.auth/admin.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Do we need to add `playwright/.auth` to the `.gitignore` like the Playwright docs suggest? - https://playwright.dev/docs/auth#core-concepts